index.html

```diff
@@ -0,0 +1,252 @@
+          class="ellipse-4"
+          alt=""
+        /><img src="./img/Ellipse-5.svg" class="ellipse-5" alt="" /><img
+          src="./img/Ellipse-8.svg"
+          class="ellipse-8"
+          alt=""
+        /><img src="./img/Ellipse-11.svg" class="ellipse-11" alt="" /><img
+          src="./img/Ellipse-12.svg"
+          class="ellipse-12"
+          alt=""
+        /><img src="./img/Ellipse-13.svg" class="ellipse-13" alt="" /><img
+          src="./img/Ellipse-14.svg"
+          class="ellipse-14"
+          alt=""
+        /><img src="./img/Ellipse-15.svg" class="ellipse-15" alt="" /><img
+          src="./img/Rectangle-4.svg"
+          class="rect-4"
+          alt=""
+        /><img src="./img/Rectangle-5.svg" class="rect-5" alt="" /><img
+          src="./img/Rectangle-6.svg"
+          class="rect-6"
+          alt=""
+        /><img src="./img/Rectangle-7.svg" class="rect-7" alt="" />
+      </div>
+      <section class="headline">
+        <h1>
+          I'm Ayoub <br />
+          Glad to see you!
+        </h1>
+        <p class="presentation">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+        <p class="connect">Let's connect</p>
+        <ul class="social-media">
+          <li class="flex-item">
+            <a href=""><img src="./img/twitter.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/Linkedin-icon.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/medium.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/github.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""
+              ><img src="./img/angellist.svg" alt="" class="flex-item"
+            /></a>
+          </li>
+        </ul>
+      </section>
+    </header>
+    <div class="works">
+      <section class="card-works">
+        <img src="./img/Snapshoot-Portfolio.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Tonic</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="submit">See Project</button>
```

Comment:
Awesome job you did with the styling states of the button 🔥🔥🔥

Reply:
Thanks